flink-yarn/src/main/java/org/apache/flink/yarn/YarnResourceManager.java

```diff
@@ -452,7 +454,7 @@ private void removeContainerRequest(AMRMClient.ContainerRequest pendingContainer
 
 	@Override
 	public void onShutdownRequest() {
-		closeAsync();
+		onFatalError(new FlinkException(ERROR_MASSAGE_ON_SHUTDOWN_REQUEST));
```

Comment:
```suggestion
		onFatalError(new ResourceManagerException(ERROR_MASSAGE_ON_SHUTDOWN_REQUEST));
```

Reply:
Thanks for your review and fix @tillrohrmann!